.github/workflows/ci.yml

```diff
@@ -284,7 +284,6 @@ jobs:
             ${{ runner.os }}-${{ env.cache-name }}-
 
       - name: Build React App
-        if: ${{ steps.cache-node-modules.outputs.cache-hit != 'true' }}
```

Comment:
What does this do?

Reply:
A request from @trent-codecov

Reply:
Basically this would stop the app from being built if we didn't have a cache hit